src/frontend/translations/translation.json

```diff
@@ -0,0 +1 @@
fermer ni recharger la page.","components.DashboardVideoPaneProgress.progressLabel":"Progression du téléversement","components.DownloadVideo.downloadVideo":"Télécharger la vidéo : ","components.DownloadVideo.480":"480p: SD (Mobile)","components.DownloadVideo.720":"720p: HD (normal)","components.DownloadVideo.1080":"1080p: FullHD (haute qualité)","components.ErrorComponent.lti.text":"Nous ne pouvons vérifier vos droits pour accéder à cette ressource. Merci de prendre contact avec votre instructeur. Si vous êtes l'instructeur, merci de vérifier vos paramètres.","components.ErrorComponent.lti.title":"Une erreur s'est produite pendant le chargement de la vidéo","components.ErrorComponent.notFound.text":"La vidéo n'existe pas ou n'a pas encore été publiée. Si vous êtes un instructeur assurez vous d'être correctement authentifié.","components.ErrorComponent.notFound.title":"La vidéo n'est pas disponible","components.ErrorComponent.policy.text":"Nous ne pouvons vérifier si vous avez les droits nécessaires pour téléverser un fichier. Vérifiez vos paramètres et/ou réessayez.","components.ErrorComponent.policy.title":"Impossible de vérifier vos droits pour téléverser le fichier","components.ErrorComponent.upload.text":"Vous pouvez réessayer dans quelques instants. Vous devriez vérifier la qualité de votre connection internet.","components.ErrorComponent.upload.title":"Erreur lors du téléversement du fichier vidéo","components.InstructorView.btnDashboard":"Aller au tableau de bord","components.InstructorView.title":"Aperçu instructeur 👆","components.TimedTextCreationForm.addTrackBtn":"Téléverser le fichier","components.TimedTextCreationForm.addTrackLabel":"Ajouter une langue","components.TimedTextCreationForm.error":"Une erreur s'est produite pendant la création du fichier.","components.TimedTextListItem.delete":"Supprimer","components.TimedTextListItem.replace":"Remplacer","components.TimedTextListItem.upload":"Téléverser","components.Transcripts.hideTranscript":"Masquer le transcript","components.Transcripts.transcriptLabel":"Choisir un transcript","components.Transcripts.transcriptSelect":"Choisir une langue","components.UploadForm.dropzoneButtonPick":"Sélectionner un fichier à téléverser","components.UploadForm.dropzoneDragText":"ou le glisser ici","components.UploadForm.linkToDashboard":"Revenir au tableau de bord","components.UploadForm.title-videos":"Téléverser une nouvelle vidéo","components.UploadForm.title-timedtexttracks-cc":"Téléverser un nouveau fichier de sous-titres codés (sourds et malentendants)","components.UploadForm.title-timedtexttracks-st":"Téléverser un nouveau sous-titres","components.UploadForm.title-timedtexttracks-ts":"Téléverser un nouveau transcript","components.UploadStatusPicker.ERROR":"Erreur","components.UploadStatusPicker.PENDING":"Absent","components.UploadStatusPicker.PROCESSING":"En cours de traitement","components.UploadStatusPicker.READY":"Disponible","components.UploadStatusPicker.UPLOADING":"Téléversement en cours"}}
```

Comment:
for the frontend it doesn't seem DRY to commit both the po and the json file... can we just commit the po and generate the json file dynamically during the build?

Reply:
And not for the backend ?

Reply:
The `compile` phase is not commit anymore.